Student/Student/Submissions/SubmissionRubric/RubricViewController.storyboard

```diff
@@ -46,23 +46,65 @@
                                     <outlet property="delegate" destination="lkh-x4-Wd8" id="j0O-aY-YFg"/>
                                 </connections>
                             </collectionView>
+                            <view hidden="YES" contentMode="scaleToFill" translatesAutoresizingMaskIntoConstraints="NO" id="ls2-gZ-h3j">
+                                <rect key="frame" x="0.0" y="292" width="375" height="83.5"/>
+                                <subviews>
+                                    <imageView clipsSubviews="YES" userInteractionEnabled="NO" contentMode="scaleAspectFit" horizontalHuggingPriority="251" verticalHuggingPriority="251" image="rubricLine" translatesAutoresizingMaskIntoConstraints="NO" id="RXI-be-h0l" customClass="IconView" customModule="Student" customModuleProvider="target">
+                                        <rect key="frame" x="167.5" y="0.0" width="40" height="40"/>
+                                        <constraints>
+                                            <constraint firstAttribute="width" constant="40" id="8uC-vN-VGQ"/>
+                                            <constraint firstAttribute="height" constant="40" id="piC-Fr-Nci"/>
+                                        </constraints>
+                                        <userDefinedRuntimeAttributes>
+                                            <userDefinedRuntimeAttribute type="string" keyPath="iconColorName" value="ash"/>
+                                            <userDefinedRuntimeAttribute type="string" keyPath="iconName" value="rubricLine"/>
+                                        </userDefinedRuntimeAttributes>
+                                    </imageView>
+                                    <label opaque="NO" userInteractionEnabled="NO" contentMode="left" horizontalHuggingPriority="251" verticalHuggingPriority="251" text="There is no rubric for this assignment" textAlignment="center" lineBreakMode="tailTruncation" baselineAdjustment="alignBaselines" adjustsFontSizeToFit="NO" translatesAutoresizingMaskIntoConstraints="NO" id="vKN-3v-yJh" customClass="DynamicLabel" customModule="Student" customModuleProvider="target">
+                                        <rect key="frame" x="52.5" y="64" width="270" height="19.5"/>
+                                        <fontDescription key="fontDescription" type="system" pointSize="17"/>
+                                        <nil key="textColor"/>
+                                        <nil key="highlightedColor"/>
+                                        <userDefinedRuntimeAttributes>
+                                            <userDefinedRuntimeAttribute type="string" keyPath="textColorName" value="ash"/>
```

Comment:
it probably doesn't matter, but you could use `textDark` instead of using `ash` directly. 🤷‍♂